src/shared/handlers/handleOpeningResultsPage.js

```diff
@@ -1,12 +1,11 @@
-import { readStorage } from "../";
+import { readStorage } from "..";
```

Comment:
The index files are a bit of a mixup. In some cases they're these kinda homes for importst. In other cases (e.g. the `shared/api/index.js`) they're actual modules. 

I'll admit, I was a bit lost when I first came across an import from '..', it didn't feel intuative. As I've said before, I don't think I'm a fan of importing from an index of imports as it feels indirect, and when you're only putting some path reference in the import, then it's even more opaque.

Anyway, not requesting a change. Just saying, it's a bit inconsistent and I find this style hard to read.

Really, this is niktpicking on what is a very good PR ☺️